airbyte-integrations/connectors/destination-clickhouse-v2/src/main/kotlin/io/airbyte/integrations/destination/clickhouse_v2/config/ClickhouseBeanFactory.kt

```diff
@@ -31,6 +32,11 @@ class ClickhouseBeanFactory {
                 .setPassword(config.password)
                 .setDefaultDatabase(config.resolvedDatabase)
                 .compressClientRequest(true)
+                // allow experimental JSON type
+                .serverSetting("allow_experimental_json_type", "1")
```

Comment:
I recall hearing that JSON is only available by request in clickhouse's hosted offering - is that still true?

(and if yes, what happens if you enable this option when connecting to an instance where JSON isn't available?)